src/EditorFeatures/Test2/GoToDefinition/GoToDefinitionTests.vb

```diff
@@ -437,6 +437,50 @@ end class
             Test(workspace)
         End Sub
 
+        <WpfFact, Trait(Traits.Feature, Traits.Features.GoToDefinition)>
```

Comment:
❓ Was this file reordered? The diff is very large and I'm not sure if I'm supposed to review it as "all new content" or "reordered content".